TSS.NET/TSS.Net/TbsDevice.cs

```diff
@@ -135,11 +134,14 @@ public override void DispatchCommand(CommandModifier active, byte[] inBuf, out b
                     return;
                 }
                 result = TpmRc.TbsUnknownError;
-                errMsg = Globs.GetResourceString("SubmitError2");
             }
-            else
+            else if (tbsRes == TbsWrapper.TBS_RESULT.BAD_PARAMETER)

```

Comment:
Just realized that this `else` is gratuitous, as the next `else` does the same for any TBS response. So just remove it.

Reply:
Fixed.